book/forms.rst

```diff
@@ -625,6 +625,21 @@ Now the form will skip your validation constraints. It will still validate
 basic integrity constraints, such as checking whether an uploaded file was too
 large or whether you tried to submit text in a number field.
 
+Handling Additional Fields
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.6
+    The ``allow_extra_fields`` option was introduced in Symfony 2.6.
+
+Usually, the Form component will complain when your submitted form contains
```

Comment:
How about this (I wanted to change from "your submitted form" to making "submit" sound more like a verb - since I understand that I "submit" data, but the definition of what a "submitted form" is, is a little less clear I think):

>
Usually, if you submit extra fields that aren't configured in your form, you'll get a
"This form should not contain extra fields." validation error (for example ...)